### PR TITLE
Feet/venue page

### PR DESCRIPTION
**Pull request by hcmgr**
NOTE: just merging this whilst it's functional, will style properly later
-venue page with image and minimal info
-pop up google map direction

### CODE REVIEW

**Comment by hcmgr** [comment]
-just merging this functional stuff, can style later

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -16,6 +17,7 @@ const routes: Routes = [
       { path: 'home', component: LandingComponent },
       { path: 'my-tickets', component: MyTicketsComponent },
       { path: 'my-account', component: MyAccountComponent },
+      { path: 'venue/:venue_id', component: VenueComponent},
```
Comment (Wo-m) [inline src/app/app-routing.module.ts]: venueId, keep it consistent

File: src/app/core/service/venue.service.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { VenueDetails } from "../model/venue-details.model";
+import { environment } from "../../../environments/environment.development";
+
+@Injectable({
+  providedIn: 'root'
+})
+export class VenueService {
+  private base_url: string = `${environment.backendBaseUrl}`
```
Comment (Wo-m) [inline src/app/core/service/venue.service.ts:10]: good

**Review by Wo-m — CHANGES_REQUESTED** [review]
can you add a router link from event details as well to venue page

File: src/app/core/service/venue.service.ts (hunk 2 above)
Comment (hcmgr) [reply src/app/core/service/venue.service.ts:10]: loves it

**Review by Wo-m — APPROVED** [review]

**Pull request merged**